listview/src/main/kotlin/com/alfresco/content/listview/tasks/TaskPriority.kt

```diff
@@ -0,0 +1,45 @@
+package com.alfresco.content.listview.tasks
+
+import android.widget.TextView
+import androidx.core.content.ContextCompat
+import com.alfresco.content.listview.R
+
+/**
+ * Marked as TaskPriority enum class
+ */
+enum class TaskPriority(val value: String) {
+    LOW("low"),
+    MEDIUM("medium"),
+    HIGH("high")
+}
+
+private fun getTaskPriority(priority: String): TaskPriority {
```

Comment:
match the cases as required and invalid cases move in else.

Reply:
Done